apps/andi/lib/andi/input_schemas/dataset_input.ex

```diff
@@ -110,26 +96,17 @@ defmodule Andi.InputSchemas.DatasetInput do
 
   defp validate_top_level_selector(%{changes: %{sourceFormat: source_format}} = changeset)
        when source_format in ["xml", "text/xml"] do
-    validate_required(changeset, [:topLevelSelector],
-      message: "Please enter a valid #{get_downcased_display_name(:topLevelSelector)}."
-    )
+    validate_required(changeset, [:topLevelSelector], message: "is required")
   end
 
   defp validate_top_level_selector(changeset), do: changeset
 
   defp validate_schema(%{changes: %{sourceType: source_type}} = changeset)
        when source_type in ["ingest", "stream"] do
     case Map.get(changeset.changes, :schema) do
-      [] ->
-        add_error(changeset, :schema, "#{get_display_name(:schema)} cannot be empty.")
-
-      nil ->
-        add_error(changeset, :schema, "Please enter a valid #{get_downcased_display_name(:schema)}.",
-          validation: :required
-        )
-
-      _ ->
-        validate_schema_internals(changeset)
+      [] -> add_error(changeset, :schema, "cannot be empty")
```

Comment:
Looks like the formatter changed these back?

Reply:
No, that was probably me.  😝 We ran the formatter again.